db/Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM postgres:13
+
+ADD ./v11.5.4_linuxx64_client.tar.gz /db2
```

Comment:
I uploaded this file to be here:
https://drive.google.com/file/d/1pe56WnWNV5P61jbuL-FL9cP1-rV9AIjx/view?usp=drive_link

Reply:
I/we can throw this onto an MT S3 bucket as well if needed